package.json

```diff
@@ -52,5 +59,9 @@
     "ts-node": "^10.9.2",
     "typescript": "^5.6.3",
     "vitest": "^2.1.3"
+  },
+  "engines": {
+    "node": ">=18",
+    "yarn": "please-use-npm"
```

Comment:
This has no impact; it's merely a safeguard for users like me who typically use Yarn.